listView.php

```diff
@@ -376,22 +441,66 @@
 
 <!-- Modal to filter lists - upon clicking this, the user can filter the tasks in their lists based on properties such as task title, importance, type of work, and due date -->
 <!-- The tasks(title and description) will be displayed within the modal itself, this can change in later iterations-->
-<div class="modal fade" id="filterList" tabindex="-1" aria-labelledby="filterList" aria-hidden="true">
-  <div class="modal-dialog">
-    <div class="modal-content" style="width:578px;">
-      <div class="modal-header">
-        <h5 class="modal-title" id="exampleModalLabel">Filter View</h5>
-        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
-          <span aria-hidden="true">&times;</span>
-        </button>
-      </div>
-      <div class="modal-body" >
-        <iframe sandbox="allow-top-navigation allow-scripts allow-forms" class="embed-responsive-item" id="filterListFrame" style="border:0; width:558px; height:700px;" src="listView.php"></iframe>
+
+<!-- SQL query to fetch listID and run a forloop to generate modals for each task list on a board -->
+<?php
+$stmt = $db->prepare('SELECT * FROM taskList WHERE boardID = ? ORDER BY `listID` ASC');
+            $stmt->execute(array($_GET['board']));       
+            $lists = $stmt->fetchAll();
+foreach ($lists as $list) {
+
+?>
+  <div class="modal fade" id="filterList<?php echo $list['listID']; ?>" tabindex="-1" aria-labelledby="filterList" aria-hidden="true">
+    <div class="modal-dialog">
+      <div class="modal-content" style="width:578px;">
+        <div class="modal-header">
+          <h5 class="modal-title" id="exampleModalLabel">Filter View</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        <div class="modal-body" >
+          <!-- <iframe sandbox="allow-top-navigation allow-scripts allow-forms" class="embed-responsive-item" id="filterListFrame" style="border:0; width:558px; height:700px;" src="listView.php"></iframe> -->
+          <form action="listView.php?board=<?php echo $board['boardID']; ?>" method="POST"> 
+              <input type="hidden" name="filterListID" value="<?php echo $list['listID']; ?>"> 
+              <br>
+              <input type="text" name="filt_name" value="" class="form-control" placeholder="Task Title" maxlength="255"/>
+              <br>
+              <input type="number" name="filt_effort" value="" class="form-control" placeholder="Importance" min = "0" max = "10" step="1"/>
+              <br>
+              <input type="text" name="filt_work" value="" class="form-control" placeholder="Type of Work" maxlength="255"/>
+              <br>
+              <input type="datetime-local" name="filt_date" value="unchanged" class="form-control" placeholder="Due Date"/>
+              <br>
+              <select name="sortTask">
```

Comment:
The inputs should come with a label rather than using placeholders. This would be better for the user to know what each input is. The 'Due Date' placeholder does not show in the current UI so the user does not know what to input there.

![Capture](https://user-images.githubusercontent.com/52054023/101232386-7369eb80-367f-11eb-8387-b6dde7be7403.PNG)

Reply:
Will add labels under the inputs similar to the add new task modal

Reply:
Looks good now. It is easy to know what each input is.
![Capture](https://user-images.githubusercontent.com/52054023/101233160-95b23800-3684-11eb-8d80-a5d63c6be8fa.PNG)